kpatch-build/kpatch-build

```diff
@@ -271,9 +271,14 @@ find_core_symvers() {
 }
 
 gcc_version_from_file() {
-	"$READELF" -p .comment "$1" | grep -m 1 -o 'GCC:.*'
+	"$READELF" -p .comment "$1" | grep -m 1 -o 'GCC:.*'| awk '{$1=$2=""; print $0}'|awk '{gsub(/^\s+|\s+$/, "");print}'
 }
 
+gcc_version_from_config() {
+    if [[ -n "$CONFIGFILE" ]]; then
+        cat "$CONFIGFILE" | grep -w "Compiler: gcc" | awk '{$1=$2=$3=$4=""; print $0}' | awk '{gsub(/^\s+|\s+$/, "");print}'
```

Comment:
I think this doesn't work with newer .config files which have CONFIG_CC_VERSION_TEXT instead.

Reply:
Sorry，I submit it again